From 19da2dda229257070aacb3a3658a84d9343a3433 Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Thu, 11 Jan 2018 13:16:17 +0100
Subject: [PATCH] 90dmsquash-live-ntfs: fix depends()

Dependencies should not include module number.

Cherry-picked from: f3e09bf1cb86248fb2fbc87d50f8d172898d3260
Resolves: #1449410
---
 modules.d/90dmsquash-live-ntfs/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90dmsquash-live-ntfs/module-setup.sh b/modules.d/90dmsquash-live-ntfs/module-setup.sh
index bc77376b..2354b840 100755
--- a/modules.d/90dmsquash-live-ntfs/module-setup.sh
+++ b/modules.d/90dmsquash-live-ntfs/module-setup.sh
@@ -8,7 +8,7 @@ check() {
 }
 
 depends() {
-    echo 90dmsquash-live
+    echo dmsquash-live
     return 0
 }
 
